src/App.tsx

```diff
@@ -1,61 +1,112 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
Consider moving `usersFromServer` and `todosFromServer` to a separate file and exporting them as constants. It will make the code more modular and reusable. 🧩